src/shardguard/core/mcp_integration.py

```diff
@@ -42,6 +42,11 @@ def __init__(self):
                 "args": [os.path.join(servers_dir, "web_server.py")],
                 "description": "Web operations with security controls",
             },
+            "nuke-operations": {
```

Comment:
I'd rather we have a style where you look into a directory and then add import all of the modules in there.   We've done things like this for other Python projects and it prevents you from needing to change the code every time you add a module.   

This can be done in a different PR, however.